### PR TITLE
Static Random Image Support

### PR DESCRIPTION
**Pull request by hasanalibalcioglu**
I think its a good option to have :)

### CODE REVIEW

**Comment by morawskim** [comment]
hello  @hasanalibalcioglu,

Yeap, I agree with you. This is a good option. I have thought about adding a new parameter to the method. Currently, the id parameter allows us to fetch a specific photo. The randomize argument append-only some random number to URL to prevent fetch image from a cache by a browser. If We add a new method, we don't need to check if the user provides randomize or static argument. This will be more clear for the user. We can move creating a query string to a separate method to prevent duplication. Also for static images, we don't need a randomize argument.

**Comment by hasanalibalcioglu** [comment]
Thank you for directives. As you can see i dont much experience with tests and contributing. Is it ok now?

File: src/PicsumProvider.php [hunk 1]
```diff
@@ -36,6 +36,29 @@ public static function picsumUrl($width = 640, $height = 480, $id = null, $rando
         return $baseUrl . $url . $queryString;
     }
 
+    public static function picsumUrlStatic($width = 640, $height = 480, $gray = false, $blur = null)
```
Comment (morawskim) [inline src/PicsumProvider.php:39]: Can you rename this method to picsumRandomUrl or something like this?
IMO the "Static" word confuse users. If we call this method 2 times we get different images.

Comment (hasanalibalcioglu) [reply src/PicsumProvider.php]: Yes you are right. I think "picsumStaticRandomUrl" makes it less confusing. It's mentioned as "Static Random" in picsums document.

**Comment by morawskim** [comment]
Thanks @hasanalibalcioglu

**Pull request merged**